src/rippled/lib/streams.js

```diff
@@ -44,6 +44,16 @@ const fetchLedger = (ledger, attempts = 0) => {
     });
 };
 
+const fetchServerInfo = () => {
+  return getServerInfo()
```

Comment:
Instead of `server info`, would it make more sense to use the `fee` request? - https://xrpl.org/fee.html#fee

Reply:
`fee` doesn't have the load factor. This is why we use `server_info` in xrpl.js for this calculation: https://github.com/XRPLF/xrpl.js/blob/main/packages/xrpl/src/sugar/getFeeXrp.ts#L39